Adjust/ADJUtil.m

```diff
@@ -932,8 +932,10 @@ + (BOOL)checkAttributionDetails:(NSDictionary *)attributionDetails {
     }
     // Apple Search Ads fields
     if ([ADJUtil contains:details key:@"iad-adgroup-id" value:@"1234567890"] &&
-        [ADJUtil contains:details key:@"iad-adgroup-name" value:@"AdgroupName"] &&
-        [ADJUtil contains:details key:@"iad-keyword" value:@"Keyword"]) {
+        [ADJUtil contains:details key:@"iad-keyword" value:@"Keyword"] && (
+            [ADJUtil contains:details key:@"iad-adgroup-name" value:@"AdgroupName"] ||
+            [ADJUtil contains:details key:@"iad-adgroup-name" value:@"AdGroupName"]
+        )) {
```

Comment:
Alternative: #523 Only checking for the new/fixed `AdGroupName` capitalization instead of both.